kafka-ui-react-app/src/components/ConsumerGroups/Details/TopicContents/TopicContents.tsx

```diff
@@ -9,24 +9,146 @@ interface Props {
   consumers: ConsumerGroupTopicPartition[];
 }
 
+type OrderByKey = keyof ConsumerGroupTopicPartition;
+interface Headers {
+  title: string;
+  orderBy: OrderByKey | undefined;
+}
+
+const TABLE_HEADERS_MAP: Headers[] = [
+  { title: 'Partition', orderBy: 'partition' },
+  { title: 'Consumer ID', orderBy: 'consumerId' },
+  { title: 'Host', orderBy: 'host' },
+  { title: 'Messages Behind', orderBy: 'messagesBehind' },
+  { title: 'Current Offset', orderBy: 'currentOffset' },
+  { title: 'End offset', orderBy: 'endOffset' },
+];
+
+const ipV4ToNum = (ip: string | undefined): number => {
```

Comment:
* `(ip?: string)`  undefined can be written like this

* one more thing here the typescript can automatically infer your return type you don't need to specify the return type.

Reply:
Updated